authors/apps/authentication/tests/test_base.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework.test import APIClient, APITestCase
+
+from ..models import User
+from ..serializers import RegistrationSerializer
+
+
+class BaseViewTest(APITestCase):
```

Comment:
@fahadmak This cannot be called `BaseViewTest` please change it to
```suggestion
class BaseTestCase(APITestCase):
```
And also move this to a` base.py` file instead of test_base.py.

Reply:
let me work on that